zpi-frontend/src/pages/Theses/AddThesisAdmin.tsx

```diff
@@ -550,7 +607,7 @@ function AddThesisPageAdmin() {
           <option value={-1}>{t('general.management.choose')}</option>
           {statuses.map((status) => (
             <option key={status.id} value={status.id}>
-              {status.name}
+              {statusPolishLabels[status.name] || status.name}
```

Comment:
Nie widze sensu robienia tu teho lub, skoro masz juz  tłumaczenia dla obu języków to je po prostu wykorzystaj. Będzie bardziej uniwersalnie jakby ktos jednka chciał zmienic wyświetlany napis.
Ogólnie ja u siebie w filtrach tematów tez potrzebowałam tych statusów, więc sobie tez to szybko zrobiłam (potem zmienię, zebysmy korzystali z jednej wersji), ale tu masz kod jakim ja zwracam odpowiednie wartości statusów:
![obraz](https://github.com/zpi-wojdan/zpi-student-project-management/assets/107646628/8910b4ee-d41f-4ed6-bb40-1eda62da3098)